docs/getting-started/rulesets.md

```diff
@@ -271,6 +272,13 @@ except:
     - info-description
 ```
 
+<!-- theme: info -->
+> As per the [RFC 6901](https://tools.ietf.org/html/rfc6901#section-3), special characters
+> `~` and `/` have to be escaped to `~0` and `~1` respectively. For instance, the location of the `get` method from a
+> `/todos` path in an openapi document `/root/here.yaml` would be expressed as `/root/here.yaml#/paths/~1todos/get`.
+> 
+> _Hint:_ Running Spectral cli with the `--format json` parameter is pretty useful to find out the path segments of each result.
```

Comment:
As pointed out by @m-mohr , added this small hint.

@philsturgeon Does the wording fit you?

Reply:
Sure